whois/parser.py

```diff
@@ -334,10 +334,11 @@ def load(domain, text):
             return WhoisUA(domain, text)
         elif domain.endswith('.kz'):
             return WhoisKZ(domain, text)
+        elif domain.endswith('.ir'):
+            return WhoisIR(domain, text)
         else:
             return WhoisEntry(domain, text)
 
-
```

Comment:
I think this introduces a style error:
https://www.python.org/dev/peps/pep-0008/#blank-lines
> Surround top-level function and class definitions with two blank lines.